source/Irrlicht/CXMeshFileLoader.cpp

```diff
@@ -17,14 +17,16 @@
 #endif
 //#define BETTER_MESHBUFFER_SPLITTING_FOR_X
 
+#define SET_ERR_AND_RETURN() do { ErrorState = true; return false; } while (0)
+

```

Comment:
I guess this is what exceptions are for.

Reply:
Indeed, however I won't be adding them to this code.